keyper/__init__.py

```diff
@@ -330,25 +331,24 @@ def list_keychains(*, domain=None):
             log.error("Failed to get keychains: %s", ex)
             raise
 
-        keychains = keychains.split("\n")
-
         # Cleanup the output format into a regular Python list of strings
-        keychains = [keychain.strip() for keychain in keychains]
-        keychains = [keychain for keychain in keychains if len(keychain) > 0]
-        keychains = [keychain[1:] for keychain in keychains]
-        keychains = [keychain[:-1] for keychain in keychains]
```

Comment:
I used the above format so that it was easier to see what was going on. What I _should_ have done was had an example as well. Which I didn't...  Either way, I think a comment explaining why we are dropping the first and last characters would be good (but that was on me, not you).

Reply:
Ya, I went back and forth with a few names but just threw in `result` as a bit of a placeholder so thank you for the feedback! I will update the naming right away.

Reply:
One idea would be to use something like `current.strip().strip('"\'')` to be explicit but I wasn't sure if it was guaranteed that there wouldn't be repeated quotes inside a keychain's name (`"''keychain"'"`?) so I just left it with the slice syntax.

Reply:
I wasn't sure about the wording of the comment but c648cfc was my first attempt, let me know if you think it should be modified.